.agent/jetpack-compose/theming/styles/SKILL.md

```
@@ -0,0 +1,220 @@
+           val styles: ComponentStyles = ComponentStyles
+       }
+   }
+   ```
+
+   <br />
+
+4. Provide extensions on `StyleScope` to reference theme tokens directly if
+   they are exposed using `CompositionLocals`. For example:
+
+
+   ```kotlin
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)